### ISSUE TITLE
[profile] Error creating/editing profiles

### ISSUE THREAD

**Issue by jabby09** [KlimaDAO/klimadao#1305]
In production, still getting an error message when editing a profile
![image](https://github.com/KlimaDAO/klimadao/assets/103535195/2c7e548b-1405-439e-b421-f8e84ec34621)

On other PR's in progress, existing profiles are not being displayed properly - this is my jabby profile that works fine in production
![image](https://github.com/KlimaDAO/klimadao/assets/103535195/404433dd-1231-469d-802c-d7547b4183f8)

**Comment by jabby09** [comment]
cc @Atmosfearful @ladytrekker

**Comment by 0xemc** [comment]
There is an issue with caching whereby changing the user description alternates between old and new values until the cache is cleared. Re-opening

**Comment by 0xemc** [comment]
Update 24/07/2023:

* The nonce error has been resolved in staging and will be deployed to production today.
* The caching issue seems to be a result of the complicated wait setup used to mutated profiles and users (`handleMutateUserUntil`). I believe this is a result of not using SWR and it's mechanisms correctly (it should not be this complicated). Once end to end types have been implemented with the API we can generate these swr hooks directly and remove a lot of this complexity. 

TLDR this ticket will be resolved once #1247 is in.

**Comment by 0xemc** [comment]
@jabby09 can you confirm that you are still receiving the error on edit of your profile? There are other issues with this user story but I would like to tackle them in a seperate ticket

**Comment by jabby09** [comment]
> @jabby09 can you confirm that you are still receiving the error on edit of your profile? There are other issues with this user story but I would like to tackle them in a seperate ticket

@0xemc yes exactly, when trying to edit my profile.

~~Steps to replicate:

1. Go to https://www.carbonmark.com/users/login
2. LOGIN (wallet or social)
3. Profile loads correctly
4. Click EDIT PROFILE
5. Change "About" section in modal (could be an edit of any part of the form)
6. Click SAVE CHANGES
7. Sign wallet signature request~~
8. and...it worked this time LOL

So never mind, it seems to be working now.

**Comment by jabby09** [comment]
@0xemc closing this one; will let you create the other tickets you referenced above
